### PR TITLE
Create entities for nomenclature items, load tables with data using m…

### PR DESCRIPTION
**Pull request by gyokhan28**
…igrations

### CODE REVIEW

File: src/main/java/com/example/b2b_opportunities/Entity/Skill.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @ManyToOne
+    @JoinColumn(name = "parent_id")
+    private Skill parent;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private Boolean assignable;
+    private String image;
```
Comment (Yordanofff) [inline src/main/java/com/example/b2b_opportunities/Entity/Skill.java:33]: Are we going to store the images as strings or blob/byte array?

File: src/main/java/com/example/b2b_opportunities/Entity/Seniority.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+@Table(name = "seniorities")
+public class Seniority {
+    @Id
+    @Column(name = "id", nullable = false)
+    private String id;
```
Comment (emiliyank) [inline src/main/java/com/example/b2b_opportunities/Entity/Seniority.java:20]: I saw that the FE team sent the IDs as strings. But I think it is better to have them as numbers. Usually for PK is better to use numbers (you can check the top 2 responses) - https://stackoverflow.com/questions/3162202/sql-primary-key-integer-vs-varchar 

If we need those S1, T1 keys, we can have another column for this. We have to discuss this with the FE team - can you write a question in the ClickUp about it?

File: src/main/java/com/example/b2b_opportunities/Entity/Skill.java (hunk 1 above)
Comment (Yordanofff) [reply src/main/java/com/example/b2b_opportunities/Entity/Skill.java:33]: All images are around 500-600Kb in total (99 icons), which is nothing really. I've already implemented storing those images in the DB. Checked prices for Amazon storage and that will be around $0.00005 per month. I will upload it as it's already done. We can modify it later if needed.

File: src/main/java/com/example/b2b_opportunities/Entity/Seniority.java (hunk 2 above)
Comment (Yordanofff) [reply src/main/java/com/example/b2b_opportunities/Entity/Seniority.java]: This is fixed now.

**Pull request merged**